layouts/default.vue

```diff
@@ -117,7 +117,8 @@ export default {
     ]),
     rasterLayer () {
       const rasterLayer = getRasterLayer()
-      rasterLayer.source.tiles = _.get(this.activeRasterData, 'tiles')
+      rasterLayer.source.tiles = [_.get(this.activeRasterData, 'url')]
+      console.log('hello')
```

Comment:
Should I re-enable the check on  commit?

Reply:
maybe...